Define Greek states in the English language

gsarig opened the pull request:
As per the states.php description, "States should be defined in English and translated native through localisation files". Greek states had been defined in Greek instead of English, so, according to the documentation, they should be adjusted accordingly. 


### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

<!-- Mark completed items with an [x] -->

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

### Changes proposed in this Pull Request:

<!-- Describe the changes made to this Pull Request and the reason for such changes. -->

As per the states.php description, "States should be defined in English and translated native through localisation files". Greek states had been defined in Greek instead of English, so, in this pull request they are being defined in English. To make sure that the correct, official names are being used, they have been taken from [Wikipedia](https://en.wikipedia.org/wiki/Administrative_regions_of_Greece).  

### How to test the changes in this Pull Request:

1.
2.
3.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

<!-- Mark completed items with an [x] -->

### Changelog entry

> Localization - Change default Greek states names to English.

gsarig committed:
Fix language of Greek states

Greek states were defined in Greek instead of English.

claudiosanches reviewed (approved):
Thanks for your help!

claudiosanches reviewed (changes requested):
Hello, thanks for your help, but note that we aren't using Wikipedia/ISO anymore, you should get the names from the CLDR list: https://github.com/unicode-org/cldr/blob/release-37/common/subdivisions/en.xml#L1974-L2000

gsarig committed:
Update states.php

Updated Greek states based on the names from the CLDR list (see: https://github.com/woocommerce/woocommerce/pull/26719#pullrequestreview-435847710)

claudiosanches committed:
Fixed "Attica" name

claudiosanches reviewed (approved):
Looks great, thanks!
